msal/changelog.txt

```diff
@@ -1,3 +1,14 @@
+2.0.1-preview
+=============
+This release contains:
+- Resolved logger issue in B2C pm sample for msal with object null reference.
+- Adding required claims to error messages.
+- Resolved issue where B2C access and refresh tokens were not cached due to missing claims in the id Token.
```

Comment:
Maybe something like "Implemented client side workaround for handling missing claims in id_token returned from B2C"...something like that  @jmprieur